### PR TITLE
Add ICS template; update README

### PR DESCRIPTION
**Pull request by cwgoes**
Add a simple template for easy copy-paste into new ICS standards.

Update README to reference ICS 1.

### CODE REVIEW

**Comment by cwgoes** [comment]
> LGTM, although I don't think all these sections under Specification should be required. Maybe we should make a remark.

So noted.

File: spec/ics-template.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+## Synopsis
+
+(high-level description of and rationale for specification)
+
+## Specification
+
+(main part of standard document - not all subsections are required)
+
+### Motivation
```
Comment (alexanderbez) [inline spec/ics-template.md]: As I'm writing the cosmos signed message spec, I'm having trouble finding content that differs from `Synopsis` and `Specification.Motivation`.

Comment (cwgoes) [reply spec/ics-template.md]: I think a more detailed justification could be useful sometimes - if not in that spec, just omit the motivation subsection.

File: spec/ics-template.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+(detailed technical specification: syntax, semantics, sub-protocols, algorithms, data structures, etc)
+
+### Backwards Compatibility
+
+(discussion of compability or lack thereof with previous standards)
+
+### Forwards Compatibility
+
+(discussion of compability or lack thereof with expected future standards)
```
Comment (alexanderbez) [inline spec/ics-template.md]:
```suggestion
(discussion of compatibility or lack thereof with expected future standards)
```

File: spec/ics-template.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+(desired characteristics / properties of protocol, effects if properties are violated)
+
+### Technical Specification
+
+(detailed technical specification: syntax, semantics, sub-protocols, algorithms, data structures, etc)
+
+### Backwards Compatibility
+
+(discussion of compability or lack thereof with previous standards)
```
Comment (alexanderbez) [inline spec/ics-template.md]:
```suggestion
(discussion of compatibility or lack thereof with previous standards)
```

**Comment by alexanderbez** [comment]
@cwgoes can we also increase all the header sizes by one? (e.g. `## Synopsis` => `# Synopsis`)

**Comment by cwgoes** [comment]
> @cwgoes can we also increase all the header sizes by one? (e.g. `## Synopsis` => `# Synopsis`)

Certainly; done.

**Comment by alexanderbez** [comment]
Great. This looks good to me!

**Comment by cwgoes** [comment]
Merging, future comments / PRs welcome.

**Pull request merged**